src/constants.ts

```diff
@@ -0,0 +1,54 @@
+import type { RenderType } from '@openmrs/esm-form-engine-lib';
+
+export const questionTypes = [
+  'control',
+  'encounterDatetime',
+  'encounterLocation',
+  'encounterProvider',
+  'encounterRole',
+  'obs',
+  'obsGroup',
+  'patientIdentifier',
+  'testOrder',
+  'programState',
+];
```

Comment:
You can get better types this way, I think:

```ts
export const questionTypes = [
  'control',
  'encounterDatetime',
  'encounterLocation',
  'encounterProvider',
  'encounterRole',
  'obs',
  'obsGroup',
  'patientIdentifier',
  'testOrder',
  'programState',
] as const;

export type QuestionType = typeof questionTypes[number];

export const renderTypeOptions: Record<Exclude<QuestionType, 'obs'>, Array<RenderType>> = {
  control: ['text', 'markdown'],
  encounterDatetime: ['date', 'datetime'],
  encounterLocation: ['ui-select-extended'],
  encounterProvider: ['ui-select-extended'],
  encounterRole: ['ui-select-extended'],
  obsGroup: ['group', 'repeating'],
  testOrder: ['group', 'repeating'],
  patientIdentifier: ['text'],
  programState: ['select'],
};
```

Now `QuestionType` can be used where you need to reference a member of `questionTypes` and will be kept in sync with that list.